source/org/zfin/sequence/ForeignDB.java

```diff
@@ -176,7 +176,7 @@ public static enum AvailableName {
         AGR_DISEASE("Alliance"),
         CZRC("CZRC"),
         PDB("PDB"),
-        RNA_CENTRAL("RNACentral"),
+        RNA_CENTRAL("RNA Central"),

```

Comment:
Had to change this so the fetch would work `getReferenceDatabase(RNA_CENTRAL, OTHER, SUMMARY_PAGE, ZEBRAFISH);`

Reply:
Looks good. I think it has to match the name column of foreign _db and not the display name.